### ISSUE TITLE
Upgrade frontend

### ISSUE THREAD

**Issue by acampsm** [Calipsoplus/calipsoplus-frontend#115]
Upgrade Angular to latest version

**Comment by aicampbell** [comment]
Hi Alex, I would rather not upgrade Angular just yet. There is a milestone in June which includes users testing the portal. I would rather wait until the milestone is completed before upgrading the frontend

**Comment by acampsm** [comment]
I have done In my local environment and I have tested. 
You say that I have to wait until June?

**Comment by aicampbell** [comment]
No, but I would prefer to wait until we have a fully stable release before we start major version upgrades.

@cloudbustinguk Hi Dan, what do you think of upgrading Angular?

**Comment by cloudbustinguk** [comment]
> No, but I would prefer to wait until we have a fully stable release before we start major version upgrades.
> 
> @cloudbustinguk Hi Dan, what do you think of upgrading Angular?

@aicampbell - We could test in dev branch - but I also agree with you that we now have other institutes bringing up their deployments and making progress. Angular 9.x might introduce changes which cause issues. Happy to test it with my deployment though if you want to bump it.

**Comment by acampsm** [comment]
OK, I'll wait for you to agree.
